src/components/heading/heading.component.js

```diff
@@ -138,40 +150,30 @@ class Heading extends React.Component {
     }
 
     return (
-      <div
-        className="carbon-heading__subheader"
-        data-element="subtitle"
-        id={this.props.subtitleId}
-      >
+      <StyledSubHeader data-element="subtitle" id={this.props.subtitleId}>
         {this.props.subheader}
-      </div>
+      </StyledSubHeader>
     );
   }
 
   /**
-   * Returns the classes for the component.
+   * Returns the separator if enabled and needed.
    *
-   * @method classes
-   * @return {String}
+   * @method separator
+   * @return {Object} JSX
    */
-  get classes() {
-    return classNames("carbon-heading", this.props.className, {
-      "carbon-heading--has-subheader": this.props.subheader,
-      "carbon-heading--has-back": this.props.backLink,
-      "carbon-heading--has-divider": this.props.divider,
-    });
+  get separator() {
+    return this.props.separator ? <StyledSeparator /> : null;
   }
 
   /**
    * Returns the separator if enabled and needed.
    *
-   * @method separator
+   * @method divider
    * @return {Object} JSX
    */
-  get separator() {
-    return this.props.separator ? (
-      <hr className="carbon-heading__separator" />
-    ) : null;
+  get divider() {
+    return this.props.divider ? <StyledDivider /> : null;
```

Comment:
We should probably be using our `Hr` component, then we wouldn't need to duplicate the styles. So rather than `<StyledDivider />` - `<Hr m="15px 0 20px" />`

Reply:
good point, changed